fe2-android/app/src/androidTest/java/com/github/dedis/popstellar/testutils/BundleBuilder.java

```diff
@@ -0,0 +1,31 @@
+package com.github.dedis.popstellar.testutils;
+
+import android.os.Bundle;
+
+import java.io.Serializable;
+
+/** Helper class to build {@link Bundle} */
+public class BundleBuilder {
```

Comment:
Is it normal that this builder in testutils is not used yet ?

Reply:
When I wrote the test framework, I needed it at some point. But I then changed the way the test was made and didn't need it anymore.

I left it because I am confident it will be useful in the future.

Reply:
Useful but unused (TM)
=> see above: prove it useful